File: sketches/java/FizzBuzz/src/Main/Format_multiples_of_3_and_5.java
package Main;

public class Format_multiples_of_3_and_5 implements INumberFormatter {

	@Override
	public boolean can_handle(int number) {

		return number%3==0 && number%5==0;
	}

	@Override
	public String handle(int number) {
		
		return "FizzBuzz";
	}
	
}

File: sketches/spring/HelloSpring/HelloSpring/FormatMultiplesOf3And5.cs
namespace HelloSpring
{
    public class FormatMultiplesOf3And5 : INumberFormatter
    {
        public bool CanHandle(int number)
        {
            return number%3 == 0 && number%5 == 0;
        }

        public string Handle(int number)
        {
            return "FizzBuzz";
        }
    }
}